e2e/testcases/hydration_test.go

```diff
@@ -170,6 +170,42 @@ func TestHydrateKustomizeComponents(t *testing.T) {
 	nt.Must(tg.Wait())
 }
 
+func TestHydrateExternalFiles(t *testing.T) {
+	nt := nomostest.New(t,
+		nomostesting.Hydration,
+		ntopts.Unstructured,
+	)
+
+	syncDirMap := map[types.NamespacedName]string{
+		nomostest.DefaultRootRepoNamespacedName: "external-files",
+	}
+
+	nt.T.Log("Add the external files root directory")
+	nt.Must(nt.RootRepos[configsync.RootSyncName].Copy("../testdata/hydration/external-files", "."))
+	nt.Must(nt.RootRepos[configsync.RootSyncName].CommitAndPush("add DRY configs to the repository"))
+
+	nt.T.Log("Update RootSync to sync from the external-files directory")
+	rs := k8sobjects.RootSyncObjectV1Beta1(configsync.RootSyncName)
+	nt.MustMergePatch(rs, `{"spec": {"git": {"dir": "external-files"}}}`)
+	nt.Must(nt.WatchForAllSyncs(nomostest.WithSyncDirectoryMap(syncDirMap)))
+
+	// Validate nomos status
+	latestCommit := nt.RootRepos[configsync.RootSyncName].MustHash(nt.T)
+	nt.Must(nt.Validate(configsync.RootSyncName, configsync.ControllerNamespace, &v1beta1.RootSync{},
```

Comment:
I think this check is redundant with the `WatchForAllSyncs` check above. Is there something specific you're trying to assert here?

Reply:
No I was just copying the general structure of the other tests which have a similar check. I went ahead and removed it.